examples/sushi/models/items.py

```diff
@@ -50,7 +50,7 @@
 @model(
     "sushi.items",
     kind=IncrementalByTimeRangeKind(time_column="ds", batch_size=30),
-    start="3 months ago",
+    start="1 week ago",
```

Comment:
why 1 week? can we only do 1 week for the historical models?

Reply:
1 week most closely matches what we were doing in the tests. We often set the start to 1 week on prod plans which doesn't really make sense because prod should never be a subset of data.